.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,151 @@
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: 1
+          HOMEBREW_NO_INSTALL_CLEANUP: 1
+        run: |
+          brew list cmake || brew install cmake
+          brew list ccache || brew install ccache
+          brew list ninja || brew install ninja
+          brew list pkgconfig || brew install pkgconfig
+          brew list qt || brew install qt
+          brew list chargepoint/xcparse/xcparse || brew install chargepoint/xcparse/xcparse
+          brew list pkg-config || brew install pkg-config
+          brew list google-cloud-sdk || brew install --cask google-cloud-sdk
+          brew list glfw3 || brew install glfw3
+          brew list libuv || brew install libuv
+          pip install ansi2html scipy
+
+      - name: Install dependencies
+        if: runner.os == 'Linux'
+        env:
+          DEBIAN_FRONTEND: noninteractive
+        run: |
+          sudo apt-get update
+          sudo apt-get dist-upgrade -y
+          sudo apt-get install -y ca-certificates git gzip ssh tar colorized-logs python3-scipy
+          sudo apt-get install -y awscli ccache clang-9 clang-format-9 clang-tidy-9 cmake doxygen fonts-noto g++-10 libc++-9-dev libc++abi-9-dev mesa-common-dev ninja-build nodejs npm pkg-config python3-bs4 python3-pip python3-requests python3-git python3-github software-properties-common valgrind xvfb zip
+          pip3 install cmake-format==0.5.5
+          sudo apt-get install -y libcurl4-openssl-dev libgl1-mesa-dev libgles2-mesa-dev libglfw3-dev libicu-dev libjpeg-turbo8-dev libpng-dev libuv1-dev zlib1g-dev
+          git clone --recursive https://github.com/google/bloaty.git /tmp/bloaty && cd /tmp/bloaty && git checkout 3cf5c3feca15 && cmake . && make -j $(nproc) bloaty && cp bloaty /usr/local/bin/ && rm -rf /tmp/bloaty && rm -rf ~/.ccache
+          sudo apt-get install -y g++-10
+          /usr/sbin/update-ccache-symlinks
+          sudo apt-get install xvfb
+
+      - name: npm ci
+        run: npm ci --ignore-scripts
+
+      - name: Prepare ccache
+        run: ccache --clear
+
+      - name: Cache ccache
+        uses: actions/cache@v2
+        env:
+          cache-name: ccache-v1
+        with:
+          path: ~/.ccache'
+          key: ${{ env.cache-name }}-${{ runner.os }}-${{ github.job }}-${{ github.ref }}-${{ github.sha }}-${{ github.head_ref }}
+          restore-keys: |
+            ${{ env.cache-name }}-${{ runner.os }}-${{ github.job }}-${{ github.ref }}-${{ github.sha }}
+            ${{ env.cache-name }}-${{ runner.os }}-${{ github.job }}-${{ github.ref }}
+            ${{ env.cache-name }}-${{ runner.os }}-${{ github.job }}
+
+      - name: Clear ccache statistics
+        run: |
+          ccache --zero-stats
+          ccache --max-size=2G
+          ccache --show-stats
+
+      - name: CMake
+        if: runner.os == 'macOS'
+        run: |
+          cmake . -B build -G Ninja -DCMAKE_OSX_DEPLOYMENT_TARGET=12.0 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=Debug -DMBGL_WITH_COVERAGE=ON
```

Comment:
Why set deployment target to such a new OS? This will prevent binaries running on older versions.

Reply:
Yah, good Q. I think this also might have to use the current OS version - will test it out.

Reply:
This is the minimum deployment target so the minimum OS we want to support. Either you should just omit it for now or figure out the minimum version this is still supposed to run on.